packages/markers/src/browser/problem/problem-widget.tsx

```diff
@@ -104,7 +104,7 @@ export class ProblemWidget extends TreeWidget {
                     <i className={severityClass}></i>
                 </div>
                 <div className='owner'>
-                    {'[' + problemMarker.owner + ']'}
+                    {'[' + problemMarker.data.source + ']'}
```

Comment:
`problemMarker.data.source || problemMarker.owner`  in the case if source is not provided

Reply:
I don't agree here: if I am correct that "owner" is a technical Identifier, we should not show it to the user. In that case "<none>" or the empty string would be appropriate.

Reply:
> "owner": that is a technical id.

Where did you get that? Before this change it was not used and used `source`, so there are client expecting it to be used.

I like the change, since it can show now more refined source, but if it is not present the PR should not break existing behaviour.

Reply:
> > "owner": that is a technical id.
> 
> Where did you get that? Before this change it was not used as a technical identifier, but as `source`, so there are client expecting it to be used.

Owner is filled from the id of the diagnostics collection (when going through the plugin API). The diagnostics id can be generated, and as such is a technical thing (think "diagnostic_collection_id_12709fajkö")

Reply:
The plugin extension is only client. There are other theia extensions which are using `ProblemManager` and they provide proper `owner`. They should work as before even in the plugin extension is not installed.

Reply:
anyway, I've tested with existing languages and it seems to work, let's give it a try

Reply:
I've changed my mind on this one: if a contribution does not set the "source" on the diagnostics, he might very well have interpreted the API in the sense that Theia did. So I think it's actually a good fallback and still people will notice the misuse of the diagnostic collection name.